app/models/comment.rb

```diff
@@ -1,4 +1,14 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  def update_comments_counter
+    comment_counter = post.comments_counter
+    if comment_counter.nil?
+      comment_counter = 1
+    else
+      comment_counter += 1
+    end
+    post.update(comments_counter: comment_counter)
+  end
```

Comment:
[OPTIONAL]
```suggestion
  after_save :update_comments_counter
  
  private
  
  def update_comments_counter
    comment_counter = post.comments_counter
    if comment_counter.nil?
      comment_counter = 1
    else
      comment_counter += 1
    end
    post.update(comments_counter: comment_counter)
  end
```

- [ ] Same thing for the `update_comments_counter` as mentioned in the previous comments :point_up_2: